doc/manuals/admin/cli.md

```diff
@@ -134,7 +134,7 @@ The list of available options is the following:
     the subscriptions cache in [this document](perf_tuning.md#subscription-cache)).
 -   **-noCache**. Disables the context subscription cache, so subscriptions searches are
     always done in DB (not recommended but useful for debugging).
--   **-notificationMode** *(Experimental option)*. Allows to select notification mode, either:
+-   **-notificationMode**. Allows to select notification mode, either:
```

Comment:
@fisuda this is a very small change, but maybe needed also in the Japanese translation.

Reply:
Noted with thanks!